Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,467 @@
+                {
+                    gameObjectToPlace = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Check if a collider is present on the GameObjectToPlace
+        /// </summary>
+        public bool ColliderPresent
+        {
+            get
+            {
+                if (GameObjectToPlace.GetComponent<Collider>() == null)
+                {
+                    return false;
+                }
+
+                return true;
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("The default distance (in meters) an object will be placed relative to the TrackedTargetType forward in the SolverHandler." +
+            "The GameObjectToPlace will be placed at the default placement distance if a surface is not hit by the raycast.")]
+        private float defaultPlacementDistance = 1.5f;
+
+        /// <summary>
+        /// The default distance (in meters) an object will be placed relative to the TrackedTargetType forward in the SolverHandler.
+        /// The GameObjectToPlace will be placed at the default placement distance if a surface is not hit by the raycast.
+        /// </summary>
+        public float DefaultPlacementDistance
+        {
+            get => defaultPlacementDistance;
+            set
+            {
+                if (defaultPlacementDistance != value)
+                {
+                    defaultPlacementDistance = value;
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Max distance to place an object if there is a raycast hit on a surface")]
+        private float maxRaycastDistance = 20.0f;
+
+        /// <summary>
+        /// The max distance to place an object if there is a raycast hit on a surface
+        /// </summary>
+        public float MaxRaycastDistance
+        {
+            get => maxRaycastDistance;
+            set => maxRaycastDistance = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component")]
+        private LayerMask[] magneticSurfaces = { UnityEngine.Physics.DefaultRaycastLayers };
+
+        /// <summary>
+        /// Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component.
+        /// </summary>
+        public LayerMask[] MagneticSurfaces
+        {
+            get => magneticSurfaces;
+            set => magneticSurfaces = value;
+        }
+
+        //[SerializeField]
+        //[Tooltip("Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you selecthe GameObjectToPlace.")]
+        //private bool isBeingPlaced = false;
+
+        /// <summary>
+        /// Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you select
```

Comment:
Super nit: Avoid "you" in comments:

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#writing-style

Reply:
"You" was removed